client/webpack.config.js

```diff
@@ -61,6 +61,7 @@ const config = {
     alias: {
       lib: path.resolve('./app/lib'),
       api: path.resolve('./app/api'),
+      surveyApp: path.resolve('./app/bundles/course/survey'),
```

Comment:
Whoops. Not sure why I thought that there would be a namespace conflict. Yeah, resolving from `course` is a better idea. Have updated to that. Thanks!